### PR TITLE
Updated member profiles

### PR DESCRIPTION
**Pull request by OmkarPh**
Signed-off-by: OmkarPh <omkarphansopkar@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by OmkarPh** [comment]
Added line-height to fix overlapping in bigger names like Jubayer, Kelechi and me
image

File: src/collections/members/jesus-lerma/index.mdx [hunk 1]
```diff
@@ -5,6 +5,9 @@ image_path: ./jesus-lerma.jpeg
 twitter: ""
```
Comment (Nikhil-Ladha) [inline src/collections/members/jesus-lerma/index.mdx:5]: an empty field can be removed

File: src/collections/members/jerrid-schilling/index.mdx [hunk 2]
```diff
@@ -2,8 +2,11 @@
 name: Jerrid Schilling
 position: Contributor - Documentation
 image_path: ./jerrid-schilling.jpg
-twitter: schillingjerrid
 github: schillingjerrid
+twitter: schillingjerrid
+linkedin: jerrid-schilling-20b06b8b
+location: Loveland, CO, USA
```
Comment (Nikhil-Ladha) [inline src/collections/members/jerrid-schilling/index.mdx:8]: State, Country is fine.

File: src/collections/members/kanishkar-j/index.mdx [hunk 3]
```diff
@@ -4,6 +4,11 @@ position: Contributor, MeshMate Emeritus
 image_path: ./kanishkar-j.jpg
 twitter: _kanishkarj_
```
Comment (Nikhil-Ladha) [inline src/collections/members/kanishkar-j/index.mdx:5]: Move 'twiter' after github.

**Review by JubayerJoy — APPROVED** [review]
LGTM, awesome work! @OmkarPh 🎈

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**